source/adapters/level_zero/command_buffer.cpp

```diff
@@ -862,9 +865,9 @@ createCommandHandle(ur_exp_command_buffer_handle_t CommandBuffer,
                                ZE_MUTABLE_COMMAND_EXP_FLAG_GLOBAL_OFFSET;
 
   auto Platform = CommandBuffer->Context->getPlatform();
-  ZE2UR_CALL(Platform->ZeMutableCmdListExt.zexCommandListGetNextCommandIdExp,
-             (CommandBuffer->ZeComputeCommandListTranslated,
-              &ZeMutableCommandDesc, &CommandId));
+  ZE2UR_CALL(
+      Platform->ZeMutableCmdListExt.zexCommandListGetNextCommandIdExp,
+      (CommandBuffer->ZeComputeCommandList, &ZeMutableCommandDesc, &CommandId));
```

Comment:
This will fail, you need to translate both handles, not just the copy above. This will break the code.